### PR TITLE
Removed WinHttpHandler from System.Net.Http

### PR DESCRIPTION
**Pull request by ManickaP**
`WinHttpHandler` now lives only in its own library.

Also removed `UseSocketsHttpHandler` property (corresponding environment var and app settings have no longer any influence).
Many functional tests for `WinHttpHadler` from System.Net.Http.FunctionalTests are suppressed now. This will be followed up by #339 which should moved them into System.Net.Http.WinHttpHandler.FunctionalTests.

Solves [41182](https://github.com/dotnet/corefx/issues/41182)

### CODE REVIEW

File: src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs [hunk 1]
```diff
@@ -77,9 +60,12 @@ protected static HttpClientHandler CreateHttpClientHandler(bool useSocketsHttpHa
             return handler;
         }
 
+        // Manicka: remove
```
Comment (davidsh) [inline src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs]: Is this a TODO item for another PR you're doing?

**Review by davidsh — APPROVED** [review]

File: src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs (hunk 1 above)
Comment (ManickaP) [reply src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs]: Removed, thanks for noticing.

**Comment by danmosemsft** [comment]
@ManickaP this is labeled breaking change. Can you please open an issue if necessary with https://github.com/dotnet/docs/issues/new?template=dotnet-breaking-change.md ?

I don't [see an existing one](https://github.com/dotnet/docs/issues?q=+is%3Aissue+label%3Abreaking-change+)

**Comment by ManickaP** [comment]
Done: https://github.com/dotnet/docs/issues/19727

**Pull request merged**